packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1465,22 +1528,27 @@ class _DropdownButtonState<T> extends State<DropdownButton<T>> with WidgetsBindi
       },
     );
 
+    if (widget._inputDecoration != null) {
+      result = InputDecorator(
+        decoration: widget._inputDecoration!,
+        isEmpty: widget._isEmpty,
+        isFocused: widget._isFocused,
+        child: result,
+      );
+    }
+
     return Semantics(
       button: true,
       child: Actions(
```

Comment:
I am not much familiar with `Actions`, but since `InkWell` already implements it, I suppose we can skip it here.

Reply:
No, it needs to be here: it's using a different action map than InkWell, and needs to handle the activate actions differently.